src/main/java/isaac/bastion/utils/BastionSettingManager.java

```diff
@@ -0,0 +1,47 @@
+package isaac.bastion.utils;
+
+import isaac.bastion.Bastion;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import vg.civcraft.mc.civmodcore.playersettings.PlayerSettingAPI;
+import vg.civcraft.mc.civmodcore.playersettings.gui.MenuSection;
+import vg.civcraft.mc.civmodcore.playersettings.impl.BooleanSetting;
+import vg.civcraft.mc.civmodcore.playersettings.impl.DisplayLocationSetting;
+
+import java.util.UUID;
+
+public class BastionSettingManager {
+
+	private BooleanSetting bsiOverlay;
+	private BooleanSetting showNoBastion;
+	private DisplayLocationSetting bsiLocation;
+
+	public BastionSettingManager() {
+		initSettings();
+	}
+
+	private void initSettings() {
+		MenuSection menu = PlayerSettingAPI.getMainMenu().createMenuSection("Bastion", "All settings related to Bastion");
+
+		bsiOverlay = new BooleanSetting(Bastion.getPlugin(), false, "Display Bastion Information", "bsiOverlay", "Shows if the block your standing on is bastioned territory.");
+		showNoBastion = new BooleanSetting(Bastion.getPlugin(), false, "Display if you are not in a bastion field", "showNoBastion", "If enabled, will display if you are not currently in a bastion field.");
```

Comment:
Change description to something more descriptive like "Should the Bastion overlay still be shown if you are not in a Bastion field"